src/rp2_common/pico_multicore/include/pico/multicore.h

```diff
@@ -158,24 +161,24 @@ bool multicore_fifo_push_timeout_us(uint32_t data, uint64_t timeout_us);
  *
  * See the note in the \ref multicore_fifo section for considerations regarding use of the inter-core FIFOs
  *
- * \return 32 bit unsigned data from the read FIFO.
+ * \return 32 bit data from the read FIFO.
  */
 uint32_t multicore_fifo_pop_blocking(void);
 
 /*! \brief Pop data from the read FIFO (data to the other core) with timeout.
  *  \ingroup multicore_fifo
  *
- * This function will block until there is data ready to be read or there timeout is reached
+ * This function will block until there is data ready to be read or the timeout is reached
  *
  * See the note in the \ref multicore_fifo section for considerations regarding use of the inter-core FIFOs
  *
  * \param timeout_us the timeout in microseconds
  * \param out the location to store the popped data if available
- * \return true if the data wash popped and a value copied into `out`, false if the timeout occurred before data could be pushed
+ * \return true if the data was popped and a value copied into `out`, false if the timeout occurred before data could be pushed
```

Comment:
"the timeout occurred before data could be pushed" -> "the timeout occurred before data could be popped"